Objects.js

```diff
@@ -0,0 +1,30 @@
+//Создать объект car, добавить к нему свойство color со значением 'черный'
+//Изменить свойство color объекта car на 'зеленый'
+//В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
+let car = {
+    color : "Черный",
+    };
+    car.color = 'Зеленый';
+car.power = function(car){
+    let EnginePower = prompt("Мощность двигателя","");
+    if (+EnginePower){
+    console.log("Мощность двигателя" + ":" + EnginePower);
```

Comment:
1) Forgotten tab.
2) You can use interpolation of strings for getting more readable code. Read more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals